app/src/main/java/com/github/se/eduverse/ui/others/setting/Setting.kt

```diff
@@ -0,0 +1,112 @@
+package com.github.se.eduverse.ui.others.setting
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.ArrowDropDown
+import androidx.compose.material3.*
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun SettingsScreen(onSaveClick: () -> Unit, onCancelClick: () -> Unit) {
+  var privacySettings by remember { mutableStateOf("") }
+  var selectedTheme by remember { mutableStateOf("Light") }
+  var selectedLanguage by remember { mutableStateOf("English") }
+  var isThemeDropdownExpanded by remember { mutableStateOf(false) }
+  var isLanguageDropdownExpanded by remember { mutableStateOf(false) }
+
```

Comment:
Using remember for state variables like privacySettings, selectedTheme, and selectedLanguage keeps things simple for now. However, if these settings need to be persisted beyond the session, consider moving them into a ViewModel in the future.